### DISCUSSION TITLE
Promoting this on YT seems impossible.

### DISCUSSION THREAD

**Discussion by spankywetfish** [ThioJoe/YT-Spammer-Purge#911] [General]
I've posted about this brilliant tool on a number of YT channels where they've been hit with scamspam, suggesting both the creator run it, and also viewers use it for reporting scamspam on their frequented channels.
Weirdly, not a single one of those posts has ever actually been published.
I started off linking to the repo, then moved to just saying its on github an lastly just to google YT-Spammer-Purge.
Not a single post mentioning this tool is showing in my history.
It's almost as if YT is filtering out any mention of it, as if they just don't want to deal with it... ¯\_(ツ)_/¯

**Comment by Firecul** [comment]
Is the comment being posted and just not showing in your history or maybe being caught in YTs spam filter it's self?

**Reply by Firecul** [reply]
Feel free to post on my [latest video](https://youtu.be/q_5RlYEYj78) if you want to test that part (I don't have to worry about it getting lost in 1000s of comments lol), I have the increased strictness turned on.
![chrome_Video_details_-_YouTube_Studio_-_Google_Chrome (3)](https://user-images.githubusercontent.com/34950371/185430958-ea9849cd-c6d4-4b47-9271-b35b2ed60c64.png)

**Reply by spankywetfish** [reply]
I just don't think the posts are being accepted, ironically probably being caught by YT's own spam filter as you suggested.
That was my own thought as well, which is why dropped links and suggested googling instead.
I did take a copy of the last post I made on the subject, I'll try posting it to your channel.
